.github/workflows/push_stata_image.yml

```diff
@@ -0,0 +1,51 @@
+name: Push Stata Image to quay
+
+on:
+  push:
+    paths:
+      - jupyter-pystata/**
+      - .github/workflows/push_stata_image.yml
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Maximize build space
+        uses: easimon/maximize-build-space@master
+        with:
+          root-reserve-mb: 30000
+          swap-size-mb: 1024
+          remove-dotnet: 'true'
+          remove-android: 'true'
+          remove-haskell: 'true'
+      - uses: actions/checkout@v2
+      - uses: prewk/s3-cp-action@master
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          SOURCE: 's3://ctds-stata/Stata17Linux64.tar.gz'
+          DEST: './jupyter-pystata/resources/'
+
+      - name: Extract branch name
+        shell: bash
+        run: echo "##[set-output name=branch;]$(echo $(echo ${GITHUB_REF#refs/heads/} | tr / _))"
```

Comment:
Is the `##` syntax right? Should it be `::`? Or is that needed due to how we're parsing out the current branch name?